react/week5/github-single-usersearch-app/src/Components/SearchResultsContainer.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+import SearchResults from './SearchResults';
+
+const SearchResultsContainer = ({ searchToken }) => {
+    const [results, setResults] = useState([]);
+    const [status, setStatus] = useState('');
+
+    useEffect(() => {
+        if (searchToken) {
+            setStatus('loading');
+
+            fetch(`https://api.github.com/search/users?q=${searchToken}`)
```

Comment:
Try to create abstractions for your ajax methods. Then put them all in one place